volontulo/frontend/scss/customization/_customization.scss

```diff
@@ -110,6 +74,53 @@ $darkgray-light:              #656D78 !default;
     }
 }
 
+// offer-thumbnails
+.offer-thumbnails {
+    .thumbnail {
+    font-size: 12px;
+        .heading-image {
+            position: relative;
+            margin-bottom: -35%;
+            &:after {
+                content: "";
+                position: absolute;
+                bottom: 0px;
+                left: 0px;
+                right: 0px;
+                max-height: 100%;
+                height: 4em;
+                background: linear-gradient(to bottom, 
+                            rgba(white, 0) 0%, 
```

Comment:
yeah, because it doesn't make any difference
`$black` and `$white` comes from bootflat, so the definitions are needed so we avoid unnecessary fixing